sql/pg_duckdb--0.2.0--0.3.0.sql

```diff
@@ -0,0 +1,9 @@
+CREATE FUNCTION @extschema@.approx_count_distinct(a anyelement)
```

Comment:
This should no be added as a function, but as a function but as an aggregate. Currently the following query fails on your test schema because of that:
```
> SELECT approx_count_distinct(a) FROM t group by b;
ERROR:  42803: column "t.a" must appear in the GROUP BY clause or be used in an aggregate function
LINE 1: SELECT approx_count_distinct(a) FROM t group by b;
```

Reply:
Thanks for the catch!!